grue.go

```diff
@@ -29,6 +31,42 @@ func add(args []string, conf *config.GrueConfig) error {
 	return conf.AddAccount(name, uri)
 }
 
+func list(args []string, conf *config.GrueConfig) {
+	const (
+		fmtShort = "%s\t%s\n"
+		fmtFull  = "%s:\n%s\n"
+	)
+	var full bool
+	var listCmd = flag.NewFlagSet("list", flag.ExitOnError)
```

Comment:
This should be ContinueOnError instead of ExitOnError, otherwise the flag package will call os.Exit() on error which doesn't handle deferred functions and the lockfile won't get cleaned up. (yes I also made the same mistake)

Reply:
Yep. I just copied what was already there. Will change it.